apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/instructorAiGenerateDrafts.sql

```diff
@@ -15,3 +15,12 @@ WHERE
   AND q.qid IS NOT NULL
 ORDER BY
   qid ASC;
+
+-- BLOCK select_drafts_by_course_id
```

Comment:
Nit:

```suggestion
-- BLOCK select_draft_questions_by_course_id
```

Reply:
Changed in fd0f052ad2ce1e73b49a4680c35f09c149548574.